db/db_impl.h

```diff
@@ -916,7 +916,11 @@ class DBImpl : public DB {
 
   // Force current memtable contents to be flushed.
   Status FlushMemTable(ColumnFamilyData* cfd, const FlushOptions& options,
-                       FlushReason flush_reason, bool writes_stopped = false);
+                       FlushReason flush_reason, bool writes_stopped = false,
```

Comment:
I'm thinking, maybe we should avoid default values of parameters here. What's your opinion?